test/web-platform-tests/to-upstream/css/cssom/getComputedStyle-shadow-inherit.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>getComputedStyle() inherits visibility</title>
```

Comment:
The title should probably mention "across shadow boundaries" or something like that.